nautobot/extras/tests/test_filters.py

```diff
@@ -924,6 +926,55 @@ def test_search(self):
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
 
 
+class JobButtonFilterTestCase(FilterTestCases.FilterTestCase):
+    queryset = JobButton.objects.all()
+    filterset = JobButtonFilterSet
+
+    @classmethod
+    def setUpTestData(cls):
+        job_buttons = (
+            JobButton.objects.create(
+                name="JobButton1",
+                text="JobButton1",
+                job=Job.objects.get(job_class_name="TestJobButtonReceiverSimple"),
+                confirmation=True,
+            ),
+            JobButton.objects.create(
+                name="JobButton2",
+                text="JobButton2",
+                job=Job.objects.get(job_class_name="TestJobButtonReceiverSimple"),
+                confirmation=False,
+            ),
+            JobButton.objects.create(
+                name="JobButton3",
+                text="JobButton3",
+                job=Job.objects.get(job_class_name="TestJobButtonReceiverComplex"),
+                confirmation=True,
+                weight=50,
+            ),
+        )
+
+        site_ct = ContentType.objects.get_for_model(Site)
+        for jb in job_buttons:
+            jb.content_types.set([site_ct])
+
+    def test_name(self):
+        params = {"name": ["JobButton1"]}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
+
+    def test_job(self):
+        params = {"job": [Job.objects.get(job_class_name="TestJobButtonReceiverSimple").pk]}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+
+    def test_weight(self):
+        params = {"weight": [50]}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
```

Comment:
Can you update these to test for multiple values in the filter? Also, we've been trying to move away from `count` assertions in filter tests and instead use `assertQuerysetEqualAndNotEmpty`. Example for `test_name`:

```py
    def test_name(self):
        params = {"name": ["JobButton1", "JobButton2"]}
        self.assertQuerysetEqualAndNotEmpty(self.filterset(params, self.queryset).qs, self.queryset.filter(name__in=["JobButton1", "JobButton2"]))
```

Reply:
Changed in latest commits